src/main/java/games/strategy/engine/random/CryptoRandomSource.java

```diff
@@ -18,38 +18,38 @@
    * converts an int[] to a byte[].
    */
   public static byte[] intsToBytes(final int[] ints) {
-    final byte[] rVal = new byte[ints.length * 4];
+    final byte[] bytes = new byte[ints.length * 4];
     for (int i = 0; i < ints.length; i++) {
-      rVal[4 * i] = (byte) (0x000000FF & ints[i]);
-      rVal[(4 * i) + 1] = (byte) ((0x000000FF & (ints[i] >> 8)));
-      rVal[(4 * i) + 2] = (byte) ((0x000000FF & (ints[i] >> 16)));
-      rVal[(4 * i) + 3] = (byte) ((0x000000FF & (ints[i] >> 24)));
+      bytes[4 * i] = (byte) (0x000000FF & ints[i]);
+      bytes[(4 * i) + 1] = (byte) ((0x000000FF & (ints[i] >> 8)));
+      bytes[(4 * i) + 2] = (byte) ((0x000000FF & (ints[i] >> 16)));
+      bytes[(4 * i) + 3] = (byte) ((0x000000FF & (ints[i] >> 24)));
     }
-    return rVal;
+    return bytes;
   }
 
   static int byteToIntUnsigned(final byte val) {
     return val & 0xff;
   }
 
   static int[] bytesToInts(final byte[] bytes) {
-    final int[] rVal = new int[bytes.length / 4];
-    for (int i = 0; i < rVal.length; i++) {
-      rVal[i] = byteToIntUnsigned(bytes[4 * i]) + (byteToIntUnsigned(bytes[4 * i + 1]) << 8)
+    final int[] ints = new int[bytes.length / 4];
+    for (int i = 0; i < ints.length; i++) {
+      ints[i] = byteToIntUnsigned(bytes[4 * i]) + (byteToIntUnsigned(bytes[4 * i + 1]) << 8)
           + (byteToIntUnsigned(bytes[4 * i + 2]) << 16) + (byteToIntUnsigned(bytes[4 * i + 3]) << 24);
     }
-    return rVal;
+    return ints;
```

Comment:
I believe this problem can be solved using the same algorithm as stated above, but with different buffer types

Reply:
Done in 52365a6.  Note that the legacy calculation is hard-coded to decode the `int`s in little endian order.